docs/components/dropdown-select/_spec.scss

```diff
@@ -0,0 +1,33 @@
+@import "components/dropdown-select/mixins/SelectControl";
+@import "components/dropdown-select/mixins/SelectFacade";
+
+.dropdown-select-spec {
+  hx-select-control {
+    /* ----- Pristine --------------- */
+    > select.mock-focus {
+      @include hxSelectControl($focused: true);
+
+      ~ hx-select {
+        @include hxSelectFacade($focused: true);
+      }
+
+      /* ----- Disabled --------------- */
+      &:disabled {
+        @include hxSelectControl(disabled, $focused: true);
+        ~ hx-select {
```

Comment:
Please add blank line between `@include` and `~ hx-select`...

```scss
&:disabled {
  @include ...

  ~ hx-select { 
    ...
  }
}
```

Reply:
Changes have been done, please review.